.circleci/config.yml

```diff
@@ -70,7 +70,7 @@ jobs:
 
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               DOCKER_IMAGE_TAG="latest"
-            elif [ "${CIRCLE_BRANCH}" == "summit-demo" ]; then
+            elif [[ "${CIRCLE_BRANCH}" =~ "^summit-demo" ]]; then
```

Comment:
lgtm. what is the reason to not require an exact match ? will there be multiple images ?